### PR TITLE
Bugfix: Fix WebGL renderer hot reload on Windows

### PR DESCRIPTION
**Pull request by WampyCakes**
## Motivation
Make hot reload for the WebGL renderer work on Windows

## Overview / Explanation for Changes
Removes watchdog observing the script and the subsequent update to the renderer. To fix it, JS changes [have been made here](https://github.com/ManimCommunity/manim-renderer/pull/12). This now adds a ScriptUpdated function to handle a request from the renderer on script change. For consistency sake, watchdog has been removed and this is used for all operating systems.

## Oneline Summary of Changes
```
- Fixed hot reload functionality for the WebGL renderer on Windows (:pr:`PR NUMBER HERE`)
```

## Testing Status
Works locally for me.

## Further Comments
I did not remove watchdog from the dependencies for Manim. I'll leave that decision to someone else. I'm not sure if we need it for anything else.

## Acknowledgements
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)

## Reviewer Checklist
- [ ] Newly added functions/classes are either private or have a docstring
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings
- [ ] The oneline summary has been included [in the wiki](https://github.com/ManimCommunity/manim/wiki/Changelog-for-next-release)

### CODE REVIEW

**Comment by eulertour** [comment]
I haven't had the chance to test this yet, but if it fails intermittently like it says in ManimCommunity/manim-renderer#12 I'd rather do this change only on Windows.

**Comment by WampyCakes** [comment]
I can't say for sure for comparison because I haven't used the hot reload with watchdog on Linux or Mac, but I believe this only fails when other things in the renderer get messed up. Like if it isn't able to serialize a response or some other such error, that's when it seems to fail and need restarted. So I'm not sure that it would fail and watchdog wouldn't. Not sure.

**Comment by eulertour** [comment]
Watchdog doesn't have to be restarted if there's a syntax error in the script, it runs again after the error is fixed

**Comment by WampyCakes** [comment]
When you had a python error, would it show in the electron console? Because it does for me and that seems to make the entire renderer need restarted (before and after this change).

**Review by eulertour — APPROVED** [review]

**Review by naveen521kk — APPROVED** [review]

File: manim/grpc/impl/frame_server_impl.py [hunk 1]
```diff
@@ -267,6 +231,7 @@ def FetchSceneData(self, request, context):
                         for scene in self.keyframes
                     ],
                 ),
+                path=f"{str(os.getcwd())}\\{str(self.input_file_path)}",
```
Comment (naveen521kk) [inline manim/grpc/impl/frame_server_impl.py:234]: Why not pathlib?

**Review by naveen521kk — CHANGES_REQUESTED** [review]

**Review by naveen521kk — APPROVED** [review]

**Pull request merged**